packages/react-examples/src/ComponentExamples.test.tsx

```diff
@@ -118,7 +125,18 @@ function getPackageAndExampleName(examplePath: string): [string, string] {
 }
 
 /** Run tests on these packages' examples */
-const includedPackages = ['react', 'react-cards', 'react-focus'];
+const includedPackages = [
+  'react',
+  'react-button',
+  'react-cards',
+  'react-checkbox',
+  'date-time',
```

Comment:
nit: Should `date-time` go at the beginning of this list so it is in alphabetical order?

Reply:
It will be in alphabetical order once date-time is renamed to `react-date-time` shortly 😆